app/views/providers/merits_task_lists/_task_list_item.html.erb

```diff
@@ -1,16 +0,0 @@
-<li class="app-task-list__item">
-  <span class="app-task-list__task-name">
-    <% formatted_name = t(".#{name}") %>
-    <% if status != :waiting_for_dependency %>
-      <%= govuk_link_to formatted_name, url, aria_describedby: "#{name}-status", "aria-label": formatted_name %>
-    <% else %>
-      <p><%= formatted_name %></p>
-    <% end %>
-  </span>
-  <%= govuk_tag(
-        text: t(".states.#{status}"),
-        colour: tag_colour,
-        classes: "app-task-list__tag",
```

Comment:
I cannot find any further reference to `app-task-list__tag`, could this be removed from the stylesheet?

Also, it may be worth checking if the custom classes removed from these deleted views can also be removed elsewhere.